### PR TITLE
fix(ingest): run each provider update on its own worker

### PR DESCRIPTION
**Pull request by petrjasek**
call `update_provider.delay` + fix an issue with `celery.loads`

### CODE REVIEW

**Comment by amagdas** [comment]
:+1:

File: superdesk/io/commands/update_ingest.py [hunk 1]
```diff
@@ -96,19 +101,16 @@ class UpdateIngest(superdesk.Command):
...
-                if provider.get('rule_set'):
-                    rule_set = superdesk.get_resource_service('rule_sets').find_one(_id=provider['rule_set'], req=None)
-
-                update_provider(provider, rule_set)
+                update_provider.delay(provider['_id'], get_provider_rule_set(provider))
 
 
 @celery.task
-def update_provider(provider, rule_set=None):
+def update_provider(provider_id, rule_set=None):
```
Comment (ioanpocol) [inline superdesk/io/commands/update_ingest.py]: If I remember well this was changed in the past by AUS from id to provider specially to use the provider settings that were when the task has been scheduled.

Comment (petrjasek) [inline superdesk/io/commands/update_ingest.py]: ok

**Comment by sivakuna-aap** [comment]
:100:

**Comment by ioanpocol** [comment]
:+1:

**Comment by petrjasek** [comment]
also fixed some issues with ftp, @akintolga @mdhaman let me know if it works for you

**Comment by akintolga** [comment]
:+1:

**Pull request merged**